openedx/core/djangoapps/content/block_structure/tests/test_signals.py

```diff
@@ -69,3 +71,14 @@ def test_course_delete(self):
             bs_manager.get_collected()
 
         self.assertFalse(is_course_in_block_structure_cache(self.course.id, self.store))
+
+
+class CourseBlocksSignalHandlerTest(ModuleStoreTestCase):
+    """
+    Tests specific to the course blocks signal handlers.
+    """
+    @patch('openedx.core.djangoapps.content.block_structure.tasks.update_course_in_cache.apply_async')
+    def test_library_update(self, mock_update):
```

Comment:
Sure, I just separated this out because it doesn't actually test the signal being sent or need any of the setup the above class does.